05_stringer/undrewb/main_test.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	want := strconv.Quote("127.0.0.1\n")
+	got := strconv.Quote(buf.String())
+
+	assert.Equal(t, got, want)
+}
+
+var stringerData = []struct {
+	name  string
+	input IPAddr
+	want  string
+}{
+	{name: "localhost",
+		input: IPAddr{127, 0, 0, 1},
+		want:  "127.0.0.1"},
```

Comment:
Cleaner:
```go
	{
		name: "localhost",
		input: IPAddr{127, 0, 0, 1},
		want:  "127.0.0.1",
	},
```

Reply:
yeah, this is strange. I'm surprised that `gofmt` doesn't complain.

Reply:
please fix.

Reply:
fixed formatting of test case data